clients/mediaconvert.go

```diff
@@ -209,6 +209,22 @@ func (mc *MediaConvert) outputVideoFiles(mcArgs TranscodeJobArgs, ourOutputBaseD
 			log.Log(mcArgs.RequestID, "invalid content length for s3 object", "ContentLength", s3Obj.ContentLength)
 			continue
 		}
+		// probe output mp4 files
+		if fileSuffix == "mp4" {
+			presigned, err := mc.s3.PresignS3(mc.s3TransferBucket.Host, key)
+			if err != nil {
+				log.Log(mcArgs.RequestID, "error creating s3 url for output files", "err", err)
+			}
+			outputVideoProbe, err := video.Probe(presigned, mcArgs.RequestID)
+			if err != nil {
+				log.Log(mcArgs.RequestID, "error probing output output files from S3", "err", err)
+			}
+			ovJSON, err := json.MarshalIndent(outputVideoProbe, "", "  ")
```

Comment:
@mjh1 - I went ahead and added the probing of output MP4s here. We'll need to format the returned info into `OutputVideo{}` to be returned to Studio. Please remove this printf stuff here that I just stuck in for testing purposes once you implement the studio integration piece.